qrencode.el

```diff
@@ -1084,6 +1084,16 @@ Commands:
         (message "No URL found at point")
       (qrencode--encode-to-buffer url))))
 
+;;;###autoload
+(defun qrencode-string (str)
+  "Encode STR.
+When called interactively, read STR with `read-string'."
+  (interactive "sString: ")
+  (let ((trimmed-str (string-trim str)))
+    (if (string-empty-p trimmed-str)
+	(user-error "No URL entered"))
```

Comment:
Nit: the user-error should probably say "Empty string" now that the function is renamed.

Reply:
Good catch.

Reply:
Just fixed -- thanks!